frontend/src/types/api-platform.ts

```diff
@@ -7,51 +7,12 @@
  * file that was distributed with this source code.
  */
 
-import { User } from '@/types/user';
 import { Fishbowl as apiPlatformFishbowl } from './api-platform/interfaces/fishbowl';
-import { faker } from '@faker-js/faker';
 
-export interface Fishbowl extends Omit<apiPlatformFishbowl, 'startDateTimeTz' | 'endDateTimeTz'> {
+export interface Fishbowl
+  extends Omit<apiPlatformFishbowl, '@id' | 'slug' | 'startDateTimeTz' | 'endDateTimeTz'> {
+  id?: string;
```

Comment:
If I don't this override , the project won't build since fishbowl.id is used in several places: https://github.com/Stooa/Stooa/blob/main/frontend/src/components/App/FishbowlList/index.tsx#L101